Tests/SwiftSyntaxBuilderTest/EnumCaseElementTests.swift

```diff
@@ -0,0 +1,32 @@
+import XCTest
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+final class EnumCaseElementTests: XCTestCase {
+  func test_enumCaseElementInit() {
+    let leadingTrivia = Trivia.garbageText("␣")
+    let string = SyntaxFactory.makeStringSegment("Hello World")
+    let segment = StringSegment(content: string)
+    let segments = StringLiteralSegments([segment])
+
+    let stringLiteralExpr: ExprBuildable = StringLiteralExpr(openDelimiter: nil,
+                                                             openQuote: SyntaxFactory.makeStringQuoteToken(),
```

Comment:
Can’t you just use `.stringQuote` which you defined in your last PR [here](https://github.com/apple/swift-syntax/blob/main/Sources/SwiftSyntaxBuilder/gyb_generated/Tokens.swift#L339)? Also further down for every use of `SyntaxFactory.make*Token`.

Since the token convenience constructors from you last PR also already have the trivia set, I think you don’t need any more calls to `with*Trivia` on them.

Reply:
It is not public. 
We could make them public and I think actually we should do that

Reply:
I didn’t realize that. I think they should definitely be public, otherwise I can’t think of what they are good for.

Reply:
Should it be in this PR or another? 
What do you prefer?

Reply:
Let’s put it in a follow-up PR before any more discussions about it come up, complicating this PR even further.